packages/@okta/vuepress-site/docs/guides/submit-app/submit/index.md

```diff
@@ -2,24 +2,37 @@
+In order for an app to be published in the OIN, you'll need to provide the following:
 
-In order for an app to be published in the Okta Integration Network, you'll need the following:
+* Development support contact information
+* Escalation contact information
+* Full, permanent test tenant provided to Okta
+
+Additionally, for SCIM app submissions:
 
 * A configuration guide explaining:
   * Supported features
   * Step-by-step instructions for setting up the integration
   * Known issues and troubleshooting suggestions
   * Customer support contact information
-* Development support contact information
-* Escalation contact information
-* Full, permanent test tenant provided to Okta
 * RunScope test suite results
 
-Okta analysts review the submission, create the customized app, and run it through Okta internal quality assurance testing. If any changes are required in your app, the analysts will request the change and you will need to resubmit the app with the updates.
+If you have any technical questions, you can submit them to <developers@okta.com> or post your questions on our [Developer Forum](https://devforum.okta.com/).
+
+When you are ready to submit your app for Okta to review, use the [OIN Manager](https://oinmanager.okta.com/) tool.
+
+After you submit your app integration, Okta analysts review the submission, create the customized app, and run it through Okta internal quality assurance testing. If we need additional information or any changes are required in your app, the analysts will reach out to you directly with the request. You will need to resubmit the app with the updates.
+
+The typical review time is two weeks, and you can track the progress of your submissions in the OIN Manager.
+
+If you need to change any attributes during the review phase, email your Okta App Analyst with the changes.
+
+Once Okta completes all internal QA testing and confirms that any requested changes have been made, the analysts publish the Okta-Verified integration directly to the OIN catalog.
+
+Have you submitted an app but haven't seen a change in the review status in the [OIN Manager](https://oinmanager.okta.com)? Send an email to <developers@okta.com>.
 
-Once Okta completes all internal QA processes and confirms that any requested changes are complete, the analysts publish the Okta-Verified app to the Okta Integration Network.
+If you have already submitted an integration to the OIN, but need to make changes, submit your application with the changes through the OIN Manager, as if your application were not listed in the OIN. When your submission is reviewed by Okta, we compare your submission against the existing customer-facing version, and update it accordingly. If there are special instructions you'd like to provide to us, add them in the Additional Instructions text box on the submission form, and we will contact you directly with any questions. Your app goes through the Okta QA process every time you add additional attributes.
```

Comment:
If you have already submitted an integration to the OIN, but need to make changes, submit your application with the changes through the OIN Manager as if your application **isn't** listed in the OIN. When your submission is reviewed by Okta, we compare your submission against the existing customer-facing version and update it accordingly. If there are special instructions you'd like to provide to us, add them in the **Additional Instructions** text box on the submission form, and we will contact you directly with any questions. Your app goes through the Okta QA process every time you add additional attributes.

removed the comma before the "and" in the second sentence also.
Bold Additional Instructions and remove the word "text"

Reply:
Changes made.